src/Apps/Order/Routes/Payment/index.tsx

```diff
@@ -21,42 +23,52 @@ export class PaymentRoute extends Component<PaymentProps> {
   render() {
     const { order } = this.props
     return (
-      <Responsive>
-        {({ xs }) => (
-          <TwoColumnLayout
-            Content={
-              <>
-                <Join separator={<Spacer mb={3} />}>
-                  <Placeholder height="68px" name="Credit Card" />
-                  <Placeholder height="20px" name="Billing/Shipping Check" />
-                  {!xs && (
-                    <Link to={`/order2/${order.id}/review`}>
-                      <Button size="large" width="100%">
-                        Continue
-                      </Button>
-                    </Link>
+      <>
+        <Row>
+          <Col>
+            <BuyNowStepper currentStep={"payment"} />
```

Comment:
I left this outside of the `Responsive` component since that's how it was before, but am happy to change if necessary.

Reply:
It's fine because the stepper is responsive itself